build/Docker/local/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.7'
+services:
+    # backend:
```

Comment:
Are we going to use this backend structure in the near future? If not, it should be deleted

Reply:
this is a work in progress...